sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -765,7 +767,7 @@ def test_remove_dependency_from_requirements(self):
     self.assertEqual(['apache_beam\n', 'avro-python3\n', 'numpy\n'],
                      sorted(lines))
 
-  def _create_file(
+  def __populate_requitements_cache_fake(
```

Comment:
nit: you probably wanted a single underscore here. double underscore is reserved for triggering name mangling, see: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles.

Reply:
Done